Parse-Dashboard/CLI/mfa.js

```diff
@@ -189,6 +189,12 @@ module.exports = {
       if (algorithm !== 'SHA1') {
         data.mfaAlgorithm = algorithm;
       }
+      if (digits !== 6) {
```

Comment:
I think it should be `!=`? In case a future code change uses a number object or something.

Reply:
`inquirier.prompt` specifies a `type`, so === is valid here. See:

https://github.com/parse-community/parse-dashboard/blob/7c56b757dfb6cd1dd48c75968137c16a31a9565e/Parse-Dashboard/CLI/mfa.js#L41-L54

Reply:
But is a strict equality comparison necessary here?

Reply:
Or maybe that strict equality comparison makes sense, because it prevents to interpret a `1` as `true`, right?

Reply:
the variable `number` will always be a `Number` as inquirer will convert CLI output to a number before passing through